app/jobs/updater.rb

```diff
@@ -0,0 +1,13 @@
+#class Updater  < ActiveJob::Base
+class Updater
+  def reschedule
+    self.class.set(wait_until: next_month).perform_later
+  end
+
+  def next_month
```

Comment:
As we may want to update sources with different frequencies, I might make that part of the interface that subclasses must implement.

So we could do something like:

```
def reschedule
  self.class.set(wait_until: next_update_time).perform_later
end

def next_update_time
   raise StandardError.new('must implement in subclass')
end
```

That way we could say, update civic weekly, but entrez every few months (or whatever).